tsinfer/cli.py

```diff
@@ -359,15 +361,16 @@ def add_recombination_arguments(parser):
         type=float,
         help="The recombination rate per unit genome",
     )
-    group.add_argument(
-        "--recombination_map",
-        default=None,
-        help=(
-            "The path to a file containing recombination rates along the chromosome "
-            "in HapMap format (see https://tskit.dev/msprime/docs/latest/api.html"
-            "#msprime.RateMap.read_hapmap for details of the format)"
-        ),
-    )
+    # Uncomment below when https://github.com/tskit-dev/tsinfer/issues/751 fixed
+    # group.add_argument(
+    #     "--recombination_map",
```

Comment:
Just something I spotted along the way: this should have been ``recombinatinon-map`` instead of ``recombination_map``.

Reply:
We also have "recombination_rate" and "mismatch_ratio": I kept underscore as these seemed like variable names ("set the `recombination_rate` variable") rather than command specifications ("do-this-thing"), but we could change them now?

Reply:
> That's the convention for POSIX CLIs

Didn't know this, thanks.

Reply:
done